packages/ModularDoctrineFilters/src/Adapter/Laravel/ModularDoctrineFiltersServiceProvider.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\ModularDoctrineFilters\Adapter\Laravel;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Str;
+use Nette\Utils\Strings;
+use ReflectionFunction;
+use Symplify\ModularDoctrineFilters\Contract\Filter\FilterInterface;
+use Symplify\ModularDoctrineFilters\Contract\FilterManagerInterface;
+use Symplify\ModularDoctrineFilters\FilterManager;
+
+/**
+ * @property \Illuminate\Foundation\Application $app
+ */
+final class ModularDoctrineFiltersServiceProvider extends ServiceProvider
+{
+    public function register()
+    {
+        $this->app->singleton(FilterManagerInterface::class, function (Application $application) {
+            return new FilterManager($application->make(EntityManagerInterface::class));
+        });
```

Comment:
Jsou to 2 rozdílné operace, jednou registruješ singleton a potom alias. Není to ve stylu:

```
$foo = new Foo
$foo->bar = 'baz'
$foo->save()
```
kde tyto řádky na sebe logicky navazují a jsou společné.

V tomto případě jde o 2 různé věci, které sami mezi sebou nemají vztah. 

To pak odděluju.